lua/lspconfig/fsautocomplete.lua

```diff
@@ -13,25 +13,42 @@ configs.fsautocomplete = {
     description = [[
 https://github.com/fsharp/FsAutoComplete
 
-Language Server for F# provded by FsAutoComplete (FSAC).
-
-Download a release of FsAutoComplete from [here](https://github.com/fsharp/FsAutoComplete/releases).
-Instructions to compile from source are found on the main repository.
+Language Server for F# provided by FsAutoComplete (FSAC).
 
 FsAutoComplete requires the [dotnet-sdk](https://dotnet.microsoft.com/download) to be installed.
 
+With dotnet-sdk installed FsAutoComplete can be installed with `dotnet tool install --global fsautocomplete`.
+
+Also it's possible build from source from [here](https://github.com/fsharp/FsAutoComplete/releases).
```

Comment:
Combine these sentences.

Instructions to compile from source are found on the main [repository](https://github.com/fsharp/FsAutoComplete/releases).